modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -65,6 +65,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also incorporates the following changes:
 
+=== The `a` elements were allowed to have non-inline child elements when the editor schema was set to `html4`.
+//#TINY-9805
+
+In previous versions of {productname}, the HTML4 editor schema, links (a elements) were permitted to have non-inline child elements. This allowed for block child elements within the links.
+
+Consequently, applying heading formatting to a link within a table caption would affect the editor view, even though the caption tag was not included in the serialized content. To address this issue, an update was made to the HTML4 schema, which now disallows block child elements within links.
+
+As a result, applying block formatting to the link content in {productname} 6.5, should no longer have any impact.
```

Comment:
```suggestion
=== The anchor element (`a`) could contain block child elements when the editor schema was set to *HTML 4*.
//#TINY-9805

In previous versions of {productname}, the anchor element — `<a>` — could have block elements added as children when the schema was set to *HTML 4*.

Consequently, applying a heading element (for example, an `<h1>` element) to a link within a table caption would affect the editor view, even though the table caption tag was not included in the serialized content.

To address this issue, an update was made to the editor’s HTML4 schema: block child elements are now not allowed in anchor tags.

As a result, applying block formatting to link content in {productname} 6.5, should no longer have any impact.
```
Copy edits, mostly to bring this into sync with the write up of TINY-9768 in DOC-2018.